public/content/demo/test-page/index.en.md

```diff
@@ -12,13 +12,15 @@ Figure With auto generated caption
 Figure With a custom caption
 </figure>
 
-<figure data-parse-mode="verse">
+<figure data-parse-mode="verse" id="verse-figure">
 Figure With a figcaption element
 In verse mode
 No less
 <figcaption>This is <strong>figure</strong> %index%</figcaption>
 </figure>
 
+See [](#verse-figure) For more details
```

Comment:
What about the use case of custom text to the anchor hook?

e.g.: `See [Illustration %index%](#verse-figure) For more details` or: `See [%OTHER_FIGURE_STRING% %index%](#verse-figure) For more details`

Reply:
We can support this, but it since <a> elements can contain nested nodes, the processing needs to be recursive. No biggie. If you insist, I can add this feature. I think that this should b developed when a real need arises and for now, you get either the standard or your own, unprocessed text.

Reply:
Agreed. Referenced it in a new issue: https://github.com/tomerlichtash/mels-loop/issues/303